consensus/src/rand/rand_gen/rand_manager.rs

```diff
@@ -364,4 +410,16 @@ impl<S: Share, D: AugmentedData, Storage: AugDataStorage<D>> RandManager<S, D, S
         }
         info!("RandManager stopped");
     }
+
+    pub fn observe_queue(&self) {
+        let queue = &self.block_queue.queue;
+        RAND_QUEUE_SIZE.set(queue.len() as i64);
+        info!(
```

Comment:
Downgrade to debug? Or maybe we only print if the queue is too large.

Reply:
removed.